recipes/poco/all/conanfile.py

```diff
@@ -61,7 +61,8 @@ class PocoConan(ConanFile):
         "Util": _PocoComponent("enable_util", True, ["Foundation", "XML", "JSON"], [], True),
         "XML": _PocoComponent("enable_xml", True, ["Foundation"], ["expat::expat"], True),
         "Zip": _PocoComponent("enable_zip", True, ["Util", "XML"], [], True),
-        "ActiveRecord": _PocoComponent("enable_active_record", True, ["Foundation", "Data"], [], True),
+        "ActiveRecord": _PocoComponent("enable_activerecord", True, ["Foundation", "Data"], [], True),
+        "ActiveRecordCompiler": _PocoComponent("enable_activerecord_compiler", True, ["Foundation", "Data"], [], True),
```

Comment:
I would not set the "default_option" to True. Like the other tools, it should be False, so the user of the recipe can opt-in instead of opt-out.

Reply:
Also, the is_lib flag should not be set for enable_activerecord_compiler as it is not a library, it is a tool (executable).

Reply:
Moreover it depends on Foundation & Util, not Data.

Reply:
Missed that. ActiveRecordCompiler actually dependes on XML (through trial and error), not Data.

Reply:
Yes, XML is missing as a direct requirement in the CMakeLists.txt.
But the problem can be alleviated by including XML as a dependency in the recipe.

Reply:
Yes,
Thank you for all reviews, all suggestions applied

Reply:
I know I wrote it as my first comment, and there is probably a reason for leaving it out, but I will still suggest that you change the first boolean for enable_activerecord_compiler (default_option) to False.
This makes the compiler tool an opt-in compilation, just like the other tools in the package.